app/src/main/java/nl/hogeschoolrotterdam/projectb/fragment/SearchFragment.java

```diff
@@ -39,7 +40,19 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         View view = inflater.inflate(R.layout.fragment_search, container, false);
 
         memories = Database.getInstance().getMemories();
-        adapter = new MemoriesAdapter(memories);
+        adapter = new MemoriesAdapter(memories, new MemoriesAdapter.OnClickListener() {
+            @Override
+            public void onItemClick(View view, Memory obj, int pos) {
+                Intent intent = new Intent(getContext(), MemoryDetailActivity.class);
+                intent.putExtra("EXTRA_SESSION_ID", obj.getId());
+                view.getContext().startActivity(intent);
+                AnalyticsUtil.selectContent(getContext(), "SearchOrList");
```

Comment:
Please replace `"SearchOrList"` with `"Search"` here.

It was always in the adapter, so we couldn't separate it at first. Now we can.